### PR TITLE
chore(ImageSource): enable nullable

### PR DESCRIPTION
**Pull request by ajpinedam**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

- Refactoring (no functional changes, no api changes)

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI/UI/Xaml/Media/ImageSource.Android.cs [hunk 1]
```diff
@@ -213,14 +212,19 @@ private static Bitmap RespectExifOrientation(ExifInterface exifInterface, Bitmap
 
 			var matrix = new Android.Graphics.Matrix();
 			matrix.PostRotate(rotationAngle);
-			bitmap = Bitmap.CreateBitmap(bitmap, x: 0, y: 0, width: bitmap.Width, height: bitmap.Height, matrix, true);
 
-			return bitmap;
+			var createdBitmap = Bitmap.CreateBitmap(bitmap, x: 0, y: 0, width: bitmap.Width, height: bitmap.Height, matrix, true);
+
+			return createdBitmap;
```
Comment (Youssef1313) [inline src/Uno.UI/UI/Xaml/Media/ImageSource.Android.cs:218]: https://developer.android.com/reference/android/graphics/Bitmap#createBitmap(android.graphics.Bitmap,%20int,%20int,%20int,%20int,%20android.graphics.Matrix,%20boolean)

<html>
<body>
<div class="devsite-table-wrapper" style="box-sizing: inherit; margin: var(--devsite-table-margin,16px 0); padding: 0px; overflow: auto; color: rgb(32, 33, 36); font-family: Roboto, &quot;Noto Sans&quot;, &quot;Noto Sans JP&quot;, &quot;Noto Sans KR&quot;, &quot;Noto Naskh Arabic&quot;, &quot;Noto Sans Thai&quot;, &quot;Noto Sans Hebrew&quot;, &quot;Noto Sans Bengali&quot;, sans-serif; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">

Returns
--
Bitmap | A bitmap that represents the specified subset of source This value cannot be null.

</div><br class="Apple-interchange-newline">
</body>
</html>Returns
[Bitmap](https://developer.android.com/reference/android/graphics/Bitmap)	A bitmap that represents the specified subset of source This value cannot be null.

Is it incorrectly annotated? Should we report to https://github.com/xamarin/xamarin-android?

File: src/Uno.UI/UI/Xaml/Media/Imaging/SvgImageSource.cs [hunk 2]
```diff
@@ -149,3 +150,4 @@ private async Task<ImageData> ReadFromStreamAsync(Stream stream, CancellationTok
 
 	internal bool UseRasterized => !double.IsNaN(RasterizePixelWidth) && !double.IsNaN(RasterizePixelHeight);
 }
+#nullable disable
```
Comment (Youssef1313) [inline src/Uno.UI/UI/Xaml/Media/Imaging/SvgImageSource.cs:153]: This shouldn't be needed

Comment (ajpinedam) [reply src/Uno.UI/UI/Xaml/Media/Imaging/SvgImageSource.cs]: Thanks

File: src/Uno.UI/UI/Xaml/Media/ImageSource.Android.cs (hunk 1 above)
Comment (ajpinedam) [reply src/Uno.UI/UI/Xaml/Media/ImageSource.Android.cs:218]: Thanks for the valuable feedback. This is a valid point 

cc: @jeromelaban

File: src/Uno.UI/UI/Xaml/Media/ImageSource.Android.cs [hunk 3]
```diff
@@ -231,17 +237,22 @@ private static async Task<Bitmap> DecodeStreamAsBitmapWithExifOrientation(Stream
...
+			{
+				return null;
+			}
+
 			var exifInterface = new ExifInterface(path);
 			return RespectExifOrientation(exifInterface, bitmap);
 		}
 
-		internal async Task<ImageData> Open(CancellationToken ct, Android.Widget.ImageView targetImage = null, int? targetWidth = null, int? targetHeight = null)
+		internal async Task<ImageData> Open(CancellationToken ct, Android.Widget.ImageView targetImage, int? targetWidth = null, int? targetHeight = null)
```
Comment (Youssef1313) [inline src/Uno.UI/UI/Xaml/Media/ImageSource.Android.cs:255]: The caller in ImageBrush passes null for `targetImage`, so I think it should be nullable. `ImageBrush` doesn't have `#nullable enable` so the build might be passing.

```suggestion
		internal async Task<ImageData> Open(CancellationToken ct, Android.Widget.ImageView? targetImage, int? targetWidth = null, int? targetHeight = null)
```

File: src/Uno.UI/UI/Xaml/Media/Imaging/SvgImageSource.cs [hunk 4]
```diff
@@ -56,7 +57,12 @@ private void OnUriSourceChanged(DependencyPropertyChangedEventArgs e)
 		{
 			UnloadImageData();
 		}
-		InitFromUri(e.NewValue as Uri);
+
+		if (e.NewValue is Uri uri)
+		{
+			InitFromUri(uri);
+		}
```
Comment (Youssef1313) [inline src/Uno.UI/UI/Xaml/Media/Imaging/SvgImageSource.cs:64]: I'd just annotate `InitFromUri` parameter as nullable and revert this change.

File: src/Uno.UI/UI/Xaml/Media/ImageSource.Android.cs (hunk 3 above)
Comment (ajpinedam) [reply src/Uno.UI/UI/Xaml/Media/ImageSource.Android.cs:255]: Thanks. Updated. This change also required updating a delegate signature.

File: src/Uno.UI/UI/Xaml/Media/Imaging/SvgImageSource.cs (hunk 4 above)
Comment (ajpinedam) [reply src/Uno.UI/UI/Xaml/Media/Imaging/SvgImageSource.cs]: Updated

File: src/Uno.UI/UI/Xaml/Media/Imaging/SvgImageSource.Android.cs [hunk 5]
```diff
@@ -53,25 +54,29 @@ private async Task<ImageData> GetSvgImageDataAsync(CancellationToken ct)
...
 					var filePath = await Download(ct, AbsoluteUri);
 
-					if (filePath == null)
+					if (filePath is null)
 					{
 						return ImageData.Empty;
 					}
 
-					using var fileStream = File.OpenRead(FilePath);
+					using var fileStream = File.OpenRead(filePath.LocalPath);
```
Comment (ajpinedam) [inline src/Uno.UI/UI/Xaml/Media/Imaging/SvgImageSource.Android.cs:79]: @MartinZikmund I updated this, but I wanted to validate it with you.

Comment (jeromelaban) [reply src/Uno.UI/UI/Xaml/Media/Imaging/SvgImageSource.Android.cs:79]: It's likely that it was a bug for this portion, and it's not been used in a while. Good catch!

**Review by jeromelaban — APPROVED** [review]
